processors/parsed_attribte.go

```diff
@@ -0,0 +1,71 @@
+package processors
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+)
+
+type contextKey struct{}
+
+// FromContext returns ParsedTracesData and updated context
+func FromContext(ctx context.Context) (context.Context, *ParsedTracesData) {
+	sAttr, ok := ctx.Value(contextKey{}).(*ParsedTracesData)
+	if ok {
```

Comment:
Should we add a log for this lack of data?

Reply:
The API is used even when creating the parsed data. If we split it into create and get then the get should maybe log warning.